opentelemetry-sdk/src/metrics/internal/sum.rs

```diff
@@ -26,10 +26,20 @@ impl<T: Number<T>> ValueMap<T> {
 impl<T: Number<T>> Aggregator<T> for ValueMap<T> {
     fn aggregate(&self, measurement: T, attrs: AttributeSet) {
         if let Ok(mut values) = self.values.lock() {
-            values
-                .entry(attrs)
-                .and_modify(|val| *val += measurement)
-                .or_insert(measurement);
+            let size = values.len();
+            match values.entry(attrs) {
+                Entry::Occupied(mut occupied_entry) => {
+                    let count = occupied_entry.get_mut();
+                    *count += measurement;
+                }
+                Entry::Vacant(vacant_entry) => {
+                    if self.check_stream_cardinality(size) {
+                        vacant_entry.insert(measurement);
+                    } else {
+                        println!("Warning: Maximum metric streams exceeded. Entry not added.");
```

Comment:
nit: This is not limiting the metric stream counts, but limiting the metric points within a given stream.

Reply:
In actual PR, the plan is still to enter a pre-fixed AttributeSet (`otel.metric.overflow`:`true`) right?